tests/unit/__snapshots__/open-api-helpers.spec.ts.snap

```diff
@@ -383,6 +417,19 @@ exports[`Open API helpers depictOptionalOrNullable() should depict ZodOptional 1
 }
 `;
 
+exports[`Open API helpers depictPipeline should depict inner schema depending on IO direction 0 1`] = `
+{
+  "nullable": true,
```

Comment:
this happens due to `coerce` and while `isNullable()` validate the schema's input, not output.
need to take it into account.

Reply:
fixed